tools/fluent/brand_names.md

```diff
@@ -0,0 +1,16 @@
+# Brand names
+
+Brand names, like Firefox or Sync, are stored as *private messages* in dedicated files, and shared across applications.
+
+Unless indicated otherwise in comments, the [current policy](https://www.mozilla.org/styleguide/communications/translation/) is that brand names can’t be:
+* Declined to adapt to grammatical case.
+* Transliterated.
+* Translated.
```

Comment:
As mentioned above, I think we should add a section here how to annotate brandnames with gender and other grammatical information to adapt messages referencing it.

Reply:
As explained, given that in Gecko you currently can't use gender or case for brand names, I think it would be counterproductive to add that information at this point.

Reply:
It can use gender.

Reply:
Makes sense, but I definitely need an example for that. Happy to had selectors and gender here, as long as we find an example that keep the brand itself untouched.

Gosh, I wish we had this discussion with legal already, instead of starting like this 😠

Reply:
I can come up with an example in Polish of French which uses a different form of the verb in the past tense for feminine brand names. The brand name itself isn't changed. Something like: `{ -brand-name } has been updated` = `{ -brand-name } a été mis/mise à jour`. What say you?

Reply:
Full example in Polish:

```properties
-brand-name = Firefox
    .gender = masculine

has-been-updated =
    { -brand-name.gender ->
        [masculine] { -brand-name } został zaktualizowany.
        [feminine] { -brand-name } została zaktualizowana.
        [neuter] { -brand-name } zostało zaktualizowane.
       *[other] Program { -brand-name } został zaktualizowany.
    }
```